my_list.rb

```diff
@@ -1,15 +1,15 @@
-require_relative "./MyEnumerable.rb";
+require_relative './my_enumerable'
 
 class MyList
-  include MyEnumerable;
+  include MyEnumerable
 
   def initialize(*list_arg)
-    @list = list_arg.flatten;
+    @list = list_arg.flatten
   end
 
   def each(&block)
-    @list.each(&block);
-    self; 
+    @list.each(&block)
+    self
   end
```

Comment:
- [ ] In this activity, we actually expect you to differently create your own `each` method from scratch instead of using the specified inbuilt array method. It should be similar to the example below:

```ruby
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```